program_intent_engagement/apps/api/v1/tests/test_views.py

```diff
@@ -298,3 +299,46 @@ def test_multiple_intents(self):
 
         self.assertIn(self.p1_intent_maybe.certainty, actual_certainties)
         self.assertIn(p2_intent_certain.certainty, actual_certainties)
+
+    def test_multiple_intents_same(self):
```

Comment:
Here's the test that I added. Lemme know if you think there's a better way to do asserts.